### ISSUE TITLE
ZeroNet and Edge are not really compatible

### ISSUE THREAD

**Issue by homersimpsons** [HelloZeroNet/ZeroNet#453]
Hello,

I've got a problem with one ZeroSite: 0list
In fact it's in My sites but when I click on, i've a blank page, just the logo. If I follow the link : [ZerLink to 0list](http://127.0.0.1:43110/186THqMWuptrZxq1rxzpguAivK3Bs6z84o/) it works well, but the link: [0list.bit](http://127.0.0.1:43110/0list.bit/) don't work. Can you add an option to configure access to site using the long, unreadable link instead of the short one.

**Comment by HelloZeroNet** [comment]
Is the other domain sites wors for you? eg http://127.0.0.1:43110/talk.zeronetwork.bit
What directories you see in plugins directory?
What blank means? Total white page or anythign visible?
Is any error message in the javascript console? (F12)

**Comment by homersimpsons** [comment]
I've got news!
This problem only append on Microsoft Edge, It doesn't append on chrome or Firefox. I think that this is a Edge related problem because webpage seems to not really refresh, and stats button (uper left) is interpret as an image and is not dragable.

> Is the other domain based sites works for you? eg http://127.0.0.1:43110/talk.zeronetwork.bit
Other websites was working, but sometimes they do the same, and it happens also with some long links like : [HelloZero long](http://127.0.0.1:43110/1HeLLo4uzjaLetFx6NH3PMwFP3qbRbTf3D).
> What directories you see in plugins directory?
- AnnounceZero
- CryptMessage
- disabled-(Bootstrapper; Dnschain; DonationMessage; Multiuser; UiPassword; Zeroname-local)
- Newsfeed
- Sidebar
- Stats
- Trayicon
- Zeroname

> What blank means? Total white page or anythign visible?
I've got on 0list page: stats icon (upper left), 0list icon (upper right) and, just below, 'more', clickable that developp 'blog' and contribute.
On ZeroWiki I've got: stats icon (upper left), Zerowiki icon (Top), 'login', 'index', 'home', 'history', 'edit' and 'Text is available under the Creative Commons Attribution-ShareAlike License. 
 You license your contributions and edits to our site under the same license.'
On ZeroMail I've got: stats icon (upper left), ZeroMail icon (upper right), 'New message', 'inbox', 'sent' (right), don't have contacts (I had one before) and 'Inbox: no message' and 'No message selected'
I think that all this page are 'base' page and need to be feed with data.

> Is any error message in the javascript console? (F12)
No error message displays
